functions.php

```diff
@@ -60,6 +66,16 @@ function _s_setup() {
 	) );
 
 	/*
+	 * Enable support for custom logo.
+	 */
```

Comment:
It would be good to add a link to https://codex.wordpress.org/Theme_Logo like we have done with other features.